src/index.js

```diff
@@ -20,15 +20,16 @@ export default (opts) => {
           return `- ${suiteName}`;
         })
         .join('\n')}\n` +
-      'Or it should be a handlebars template which defines and sets up your tests (see README.md)\n\n'
+      'Or it should be a handlebars template which defines and sets up your tests (see README.md)\n\n',
     );
   }
 
   const options = Object.assign(
     {
       optsEslint: null,
+      maxWarnings: -1,
```

Comment:
But if anyone is reading the docs on the API opts ([here](http://eslint.org/docs/developer-guide/nodejs-api)), they'd be confused and/or misled, because it's not an option supported by eslint's interface. We're explicitly adding synthetic support for it in this library.